assets/js/modules/ads/pax/services.test.js

```diff
@@ -93,6 +107,44 @@ describe( 'PAX partner services', () => {
 					).toEqual( [] );
 				} );
 			} );
+
+			describe( 'getPageViewConversionSetting', () => {
+				it( 'should hold correct value for enablePageViewConversion property', async () => {
+					const pageViewConversionSetting =
+						await services.conversionTrackingService.getPageViewConversionSetting();
+
+					expect(
+						pageViewConversionSetting.enablePageViewConversion
+					).toBeTruthy();
```

Comment:
We can be more specific about this since a different truthy value may actually be problematic. It's coming from our own function though so this is fine.
```suggestion
					).toBe( true );
```